Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml

```diff
@@ -1036,7 +1036,7 @@
   id: SyringeStimulants
   components:
   - type: Label
-    currentLabel: reagent-name-hyperzine
+    currentLabel: hyperzine
```

Comment:
```suggestion
    currentLabel: reagent-name-stimulants
```
You are removing the localization and hardcoding the english name, which is bad for forks in other languages.
Use the correct localization string instead. Stimulants got renamed into hyperzine at some point, but the loc string stayed the same.

Reply:
but all the other things in the file use the same system, does it still use the localisation there?

Reply:
The loc string was changed, but the ID is still the same as before
![grafik](https://github.com/user-attachments/assets/d73a3553-9949-4049-b61c-8c095828db5b)

Reply:
as long as it gets fixed! how long has it been like this anyways?